### DISCUSSION TITLE
Incremantal modes not working for Zendesk Sunshine source

### DISCUSSION THREAD

**Discussion by WisamulHaque** [airbytehq/airbyte#34085] [General]
Description:
Incremental modes are replicating data in destination even when no new data is added in source or any update is made in existing data

At Destination:

![image](https://github.com/airbytehq/airbyte/assets/106967803/5e858108-4139-4633-ac2c-b1117b865b50)

At Source Side:

![image](https://github.com/airbytehq/airbyte/assets/106967803/353e5870-9d56-46bd-a5c4-c0fa50a0a571)
![image](https://github.com/airbytehq/airbyte/assets/106967803/e3b92df5-a474-49a4-a0e7-098ce307edeb)
![image](https://github.com/airbytehq/airbyte/assets/106967803/f31dc56d-9cef-41c2-9343-5937c53ef5e9)

Logs:
[ZendeskSunshineLogs.txt](https://github.com/airbytehq/airbyte/files/13885426/ZendeskSunshineLogs.txt)

**Comment by sajarin** [comment]
hey @WisamulHaque this might be intended behavior with incremental | append, are you able to use incremental | deduped?

[a comment by an automated account is not shown]
